tests/core/test_ensemble.py

```diff
@@ -184,152 +174,6 @@ def test_policy_priority():
     assert prediction.probabilities == priority_2_result.probabilities
 
 
-def test_fallback_mapping_restart():
-    domain = Domain.load("data/test_domains/default.yml")
-    events = [
-        ActionExecuted(ACTION_DEFAULT_FALLBACK_NAME, timestamp=1),
-        utilities.user_uttered(USER_INTENT_RESTART, 1, timestamp=2),
-    ]
-    tracker = DialogueStateTracker.from_events("test", events, [])
-
-    two_stage_fallback_policy = TwoStageFallbackPolicy(
-        priority=2, deny_suggestion_intent_name="deny"
-    )
-    mapping_policy = MappingPolicy(priority=1)
-
-    mapping_fallback_ensemble = SimplePolicyEnsemble(
-        [two_stage_fallback_policy, mapping_policy]
-    )
-
-    prediction = mapping_fallback_ensemble.probabilities_using_best_policy(
-        tracker, domain, RegexInterpreter()
-    )
-    index_of_mapping_policy = 1
-    next_action = rasa.core.actions.action.action_for_index(
-        prediction.max_confidence_index, domain, None
-    )
-
-    assert (
-        prediction.policy_name
-        == f"policy_{index_of_mapping_policy}_{MappingPolicy.__name__}"
-    )
-    assert next_action.name() == ACTION_RESTART_NAME
-
-
-@pytest.mark.parametrize(
-    "events",
-    [
-        [
-            ActiveLoop("test-form"),
-            ActionExecuted(ACTION_LISTEN_NAME),
-            utilities.user_uttered(USER_INTENT_RESTART, 1),
-        ],
-        [
-            ActionExecuted(ACTION_LISTEN_NAME),
-            utilities.user_uttered(USER_INTENT_RESTART, 1),
-        ],
-    ],
-)
-def test_mapping_wins_over_form(events: List[Event]):
```

Comment:
Just reading the names of the tests being removed, we might want to create a test case that checks RulePolicy in this way, right?

Reply:
I'm not sure. This is comparing `MappingPolicy` against `FormPolicy`, neither of which exist anymore. Not sure what I would be comparing `RulePolicy` to in this case?